js/main.js

```diff
@@ -13,6 +13,12 @@ var avatarNumbers = [1, 2, 3, 4, 5, 6, 7, 8];
 var placeType = ['palace', 'flat', 'house', 'bungalo'];
 var mapBlock = document.querySelector('.map');
 var pinList = document.querySelector('.map__pins');
+var inputPrice = document.querySelector('#price');
+var selectorType = document.querySelector('#type');
```

Comment:
_selector_, лишнее во всех переменных

```suggestion
var fieldType = document.querySelector('#type');
```